index.bs

```diff
@@ -581,6 +595,26 @@ To <dfn>handle an incoming message</dfn> given a [=WebSocket connection=]
 
 </div>
 
+<div algorithm>
+To <dfn export>emit an event</dfn> given |body| and |related contexts|:
+
+ 1. [=Assert=]: |body| has size 2 and contains "<code>method</code>" and "<code>params</code>".
+
+ 1. If the [=current session=] is null, or the [=current session=]'s [=WebSocket
```

Comment:
"current session" is used elsewhere without a null check, is it needed here?

Reply:
Yep, you're right. A notable case is https://w3c.github.io/webdriver-bidi/#object-id-for-an-object, but that's currently unreachable. We'll have to deal with this when we have an equivalent of https://w3c.github.io/webdriver/#dfn-execute-async-script. Nothing is broken right now though, AFAICT.